app/javascript/components/Admin/Users/PayoutInfo/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { cast } from "ts-safe-cast";
+
+import { request } from "$app/utils/request";
+
+import PayoutInfo, { type PayoutInfoProps } from "$app/components/Admin/Users/PayoutInfo/PayoutInfo";
+import type { User } from "$app/components/Admin/Users/User";
+
+type AdminUserPayoutInfoProps = {
+  user: User;
+};
+
+const AdminUserPayoutInfo = ({ user }: AdminUserPayoutInfoProps) => {
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [data, setData] = React.useState<PayoutInfoProps | null>(null);
+  const [isVisible, setIsVisible] = React.useState(false);
+  const elementRef = React.useRef<HTMLDivElement>(null);
+
+  React.useEffect(() => {
+    if (!elementRef.current) return;
+    const observer = new IntersectionObserver((entries) => setIsVisible(entries.some((entry) => entry.isIntersecting)));
+    observer.observe(elementRef.current);
+    return () => observer.disconnect();
+  }, [isVisible]);
```

Comment:
Not in love with how this looks, but functionally it's the best of both worlds IMO - key info is displayed inline like in current admin, but we don't load everything up front on pages with many users like the refund queue